[rospy.client][INFO] 2020-09-01 10:19:15,803: init_node, name[/diagnostic_move_base_status], pid[29906]
[xmlrpc][INFO] 2020-09-01 10:19:15,803: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-09-01 10:19:15,804: Started XML-RPC server [http://cuboid:36629/]
[rospy.impl.masterslave][INFO] 2020-09-01 10:19:15,804: _ready: http://cuboid:36629/
[rospy.init][INFO] 2020-09-01 10:19:15,806: ROS Slave URI: [http://cuboid:36629/]
[rospy.registration][INFO] 2020-09-01 10:19:15,806: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-09-01 10:19:15,806: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-09-01 10:19:15,907: registered with master
[rospy.rosout][INFO] 2020-09-01 10:19:15,908: initializing /rosout core topic
[rospy.rosout][INFO] 2020-09-01 10:19:15,915: connected to core topic /rosout
[rospy.simtime][INFO] 2020-09-01 10:19:15,918: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-09-01 10:19:16,102: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-09-01 10:19:16,204: topic[/diagnostics] adding connection to [/diag_agg], count 0
[rospy.internal][INFO] 2020-09-01 10:19:16,737: topic[/move_base/result] adding connection to [http://cuboid:41905/], count 0
[rospy.core][INFO] 2020-09-01 10:25:39,419: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-09-01 10:25:39,423: topic[/diagnostics] removing connection to /diag_agg
[rospy.internal][INFO] 2020-09-01 10:25:39,424: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-09-01 10:25:39,424: topic[/move_base/result] removing connection to http://cuboid:41905/
[rospy.internal][WARNING] 2020-09-01 10:25:39,425: Unknown error initiating TCP/IP socket to cuboid:49883 (http://cuboid:41905/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-09-01 10:25:39,425: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2020-09-01 10:25:39,425: signal-2
[rospy.core][INFO] 2020-09-01 10:25:39,533: signal_shutdown [atexit]
